lib/api/api.js

```diff
@@ -372,4 +372,14 @@ Api.prototype.setMocks = function (val, store, pdp, storeconn, pep) {
   if (pep)
     this.pep = pep;
 };
+
+Api.prototype.getEntityPolicies = function (entity_id, entity_type) {
+  //TODO who is able to read the policices?
+  return this.pap.getEntityPolicies(entity_id, entity_type);
+};
+
+Api.prototype.setEntityPolicy = function (entity_id, entity_type, property, policy) {
+  return this.pap.setEntityPolicy(entity_id, entity_type, property, policy);
+};
```

Comment:
In these two calls we need the auth_result argument specifying the user who is executing the call. This auth_result contains information about the user who is executing the action. So, this will be used in the future to validate wheter or not he can get or set the entity's policies.